ghost/admin/tests/acceptance/whats-new-test.js

```diff
@@ -0,0 +1,358 @@
+            url: 'https://ghost.org/changelog/new-feature',
+            featured: true
+        },
+        old: {
+            title: 'Old Update',
+            custom_excerpt: 'This is old',
+            published_at: '2018-12-01T12:00:00.000+00:00',
+            url: 'https://ghost.org/changelog/old-feature',
+            featured: true
+        },
+        newNonFeatured: {
+            title: 'Non-Featured Update',
+            custom_excerpt: 'This is not featured',
+            published_at: '2024-01-15T12:00:00.000+00:00',
+            url: 'https://ghost.org/changelog/regular-feature',
+            featured: false
+        },
+        newRegular: {
+            title: 'New Update',
+            custom_excerpt: 'New feature',
+            published_at: '2024-01-15T12:00:00.000+00:00',
+            url: 'https://ghost.org/changelog/new',
+            featured: false
+        },
+        latest: {
+            title: 'Latest Update',
+            custom_excerpt: 'Latest feature',
+            published_at: '2024-01-15T12:00:00.000+00:00',
+            url: 'https://ghost.org/changelog/latest',
+            featured: true,
+            feature_image: 'https://ghost.org/image1.jpg'
+        },
+        previous: {
+            title: 'Previous Update',
+            custom_excerpt: 'Previous feature',
+            published_at: '2024-01-10T12:00:00.000+00:00',
+            url: 'https://ghost.org/changelog/previous',
+            featured: false
+        },
+        latestNonFeatured: {
+            title: 'Latest Update',
+            custom_excerpt: 'Latest feature',
+            published_at: '2024-01-15T12:00:00.000+00:00',
+            url: 'https://ghost.org/changelog/latest',
+            featured: false
+        }
+    };
+
+    // Helper to mock changelog endpoint
+    function mockChangelog(server, entries) {
+        server.get('https://ghost.org/changelog.json', function () {
+            return {
+                posts: entries,
+                changelogUrl: 'https://ghost.org/changelog/'
+            };
+        });
+    }
+
+    beforeEach(async function () {
+        // Create default user for most tests
+        const role = this.server.create('role', {name: 'Owner'});
+        this.user = this.server.create('user', {
+            id: '1',
+            roles: [role],
+            accessibility: JSON.stringify({
+                whatsNew: {
+                    lastSeenDate: '2019-01-01 00:00:00'
+                }
+            })
+        });
+
+        await authenticateSession();
+    });
+
+    describe('toast notification', function () {
+        let toast;
+
+        beforeEach(function () {
+            toast = new WhatsNewToast();
+        });
+
+        it('shows toast with latest entry when there are new featured entries', async function () {
+            mockChangelog(this.server, [ENTRIES.newFeatured]);
+
+            await visit('/site');
+            await settled();
```

Comment:
We shouldn't need settled() in almost all cases, the ember test helpers call this internally before resolving so we can clean these extraneous uses up 🙂 If we find they are needed to make a test pass it suggests we have a deeper issue somewhere.